src/common/layouts/Header/Header.scss

```diff
@@ -3,7 +3,14 @@
   position: fixed;
   width: 100%;
 
-  .menu {
-    line-height: 64px;
+  .nav-item-container {
+    .nav-item {
+      color: hsla(0, 0%, 100%, 0.65);
```

Comment:
why not ``rgba``?

Reply:
`hsl` is better format for making darker/lighter colors in general but we could use rgba, I don't think it matters.